jwql/tests/test_data_containers.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+"""Tests for the ``data_containers`` module in the ``jwql`` web
+application.
+
+Authors
+-------
+
+    - Matthew Bourque
+
+Use
+---
+
+    These tests can be run via the command line (omit the -s to
+    suppress verbose output to stdout):
+
+    ::
+
+        pytest -s test_data_containers.py
+"""
+
+import glob
+import os
+
+import pytest
+
+# Skip testing this module if on Jenkins
+ON_JENKINS = '/home/jenkins' in os.path.expanduser('~')
+try:
+    from jwql.website.apps.jwql import data_containers
+    from jwql.utils.utils import get_config
+except:
+    pass
+
+
+@pytest.mark.skipif(ON_JENKINS, reason='Requires access to central storage.')
+def test_get_acknowledgements():
+    """Tests the ``get_acknowledgements`` function."""
+
+    acknowledgements = data_containers.get_acknowledgements()
+    assert isinstance(acknowledgements, list)
+    assert len(acknowledgements) > 0
+
+
+@pytest.mark.skipif(ON_JENKINS, reason='Requires access to central storage.')
+def test_get_all_proposals():
+    """Tests the ``get_all_proposals`` function."""
+
+    proposals = data_containers.get_all_proposals()
+    assert isinstance(proposals, list)
+    assert len(proposals) > 0
+
+
+@pytest.mark.xfail
+def test_get_dashboard_components():
+    """Tests the ``get_dashboard_components`` function."""
+
+    dashboard_components, dashboard_html = data_containers.get_dashboard_components()
+    assert isinstance(dashboard_components, dict)
+    assert isinstance(dashboard_html, dict)
```

Comment:
Do we care about testing if the dictionary has stuff in it or not or is testing if it exists enough?

Reply:
There's a couple other times in here where we test for a dict but not if it's empty or not - if we decide to change that here we'd want to do that in the other tests too.

Reply:
@catherine-martlin Good idea! I can add those checks, they would be nice to have.